### DISCUSSION TITLE
Git info

### DISCUSSION THREAD

**Discussion by AmitMPawar** [AbhyudayaSharma/react-git-info#30] [Q&A] [answered]
Wanted the author's name and email info.

Thanks, 
Amit

**Comment by AbhyudayaSharma** [comment, marked as answer]
@AmitMPawar Feel free to create a PR!

**Comment by AmitMPawar** [comment]
Hello Abhyudaya,

Sorry I couldn’t understood the meaning 
> Closed #30 as resolved
> 
Please can I know the meaning 


Thank you
Amit Pawar
